spec/quality_spec.rb

```diff
@@ -40,6 +40,22 @@ def check_for_extra_spaces(filename)
     "#{filename} has spaces on the EOL on lines #{failing_lines.join(", ")}"
   end
 
+  def check_for_expendable_words(filename)
+    failing_line_message = []
+    useless_words = /\b(actually|obviously|just|clearly|basically|really)\b/i
```

Comment:
Does this handle capitalized instances of these words? (i.e. "Actually, the best to do this is [...]")